docs/chat-configuration/ui-customization/ui-customization.md

```diff
@@ -9,3 +9,97 @@ nav_order: 3
+| `MultipleSelectionConfiguration`   | titleConfiguration, persistentOptionConfiguration                                                                                           |
+| `IncomingLiveConfiguration`  | backgroundColor, backgroundImage, textColor, customFont,avatar, avatarPosition, borderRadius                                             |
+| `OutgoingConfiguration`      | backgroundColor, backgroundImage, textColor, customFont,avatar, avatarPosition, borderRadius, sentSuccessfullyIcon, sentFailureIcon, pendingIcon |
+| `CarouselConfiguration`      | backgroundColor, backgroundImage, customFont, avatar, textColor                                                     |
+| `SystemMessageConfiguration` | backgroundColor, backgroundImage, textColor, customFont, borderRadius        
+| `ChatBarConfiguration` | backgroundColor, font, textColor, image, agentName, endChatBtnTitle, endChatBtnTextColor, enabled, userTranscriptBtnImage
+| `BoldFormConfiguration` | titleConfig |
+| `BoldFormElementConfiguration` | backgroundColor, font, textColor |
+
+
+### How To Set Configuration
+
+In following sample we will customize chat view.
+
+```swift
+self.chatController.viewConfiguration.chatViewConfig.hyperlinkColor = UIColor.blue
+self.chatController.viewConfiguration.chatViewConfig.dateStamp.textColor = UIColor.red
+self.chatController.viewConfiguration.chatViewConfig.dateStamp.customFont = CustomFont(font: UIFont.systemFont(ofSize: 20))
+self.chatController.viewConfiguration.chatViewConfig.timeStamp.textColor = UIColor.yellow
+self.chatController.viewConfiguration.chatViewConfig.timeStamp.customFont = CustomFont(font: UIFont.systemFont(ofSize: 30))
+self.chatController.viewConfiguration.chatViewConfig.avatarSize = CGSize(width: 50, height: 50)
+```
+
+----
+
+## Avatar Image 
+
+Should use images in the size to be displayed for the client.
+
+To update the image:
+
+### Default
+Same image will be shown for different appearances (Dark/ Light).
+```swift
+self.chatController.viewConfiguration.incomingBotConfig.avatar = UIImage(named: "robot")
+```
+Or
+### Support Different Appearances
+```swift
+self.chatController.viewConfiguration.incomingBotConfig.avatrImageName = "robot"
+```
+>Note: To support different appearances make sure image exist in [Application Bundle](https://developer.apple.com/documentation/uikit/uiimage/providing_images_for_different_appearances).
+
+
+## Avatar Positioning 
+
+`ChatElementConfiguration` has a property `avatarPosition` of type `AvatarPosition` the default value for outgoing element is `AvatarPositionBottomLeft` and for incoming element is `AvatarPositionBottomRight` for changing the position:
+>Note: The shape of the bubble corners adjacent to the `avatarPosition`, will be square shaped.
+
+```swift
+self.chatController.viewConfiguration.incomingBotConfig.avatarPosition = .topLeft
+```
+
+### Avatar Position Options
+
+``` Objective C
```

Comment:
there is issue with this code block please see:
https://logmein-bold-mobile.github.io/bold360-mobile-docs-ios/docs/chat-configuration/ui-customization#avatar-position-options